### ISSUE TITLE
Can I pipe the Stdout?

### ISSUE THREAD

**Issue by riggerthegeek** [FormidableLabs/nodejs-dashboard#29]
&lt;praise&gt;Splendid package, well done&lt;/praise&gt;

My application uses [Bunyan](https://github.com/trentm/node-bunyan) for logging. In order to make the output more readable, my original command is `node ./app.js | bunyan`.

When I change my command to `nodejs-dashboard node ./app.js | bunyan`, I just get the message "Waiting for client connection on 9838..." and nothing else.

If I get rid of the `| bunyan` pipe, it works fine (but with the stdout an unreadable mess).

Any ideas how to get my output piped?

**Comment by jasonwilson** [comment]
Hi @riggerthegeek not sure that this setup will work in its current incarnation. You might be able to use bunyan `-p` option with some bash hackery and launch bunyan via the dashboard.

**Comment by riggerthegeek** [comment]
Yeah, that might work. It's a little bit coupled to Bunyan though.

It might be a useful addition in future to be able to pipe the output
